### PR TITLE
Demo of dropzone field

### PR DESCRIPTION
**Pull request by jorgetwgroup**
### BEFORE - What was wrong? What was happening before this PR?

Nothin wrong, just we dont have a dropzone field

### AFTER - What is happening after this PR?

Now we have a dropzone prototype

### How did you achieve that, in technical terms?

We use [Dropzone](https://www.dropzone.dev/) library to create our Pro field

### Is it a breaking change or non-breaking change?

No breaking, but we need to run npm to install library and we depend on this PR's:
https://github.com/Laravel-Backpack/PRO/pull/125
https://github.com/Laravel-Backpack/CRUD/pull/4922

### How can we test the before & after?

??

### CODE REVIEW

**Comment by tabacitu** [comment]
For future us, what needs to be done to have all needed PRs:
```bash
# clone the demo
gh pr checkout 470
cd vendor/backpack/crud && gh pr checkout 4922
cd ../pro && gh pr checkout 125
cd ../../..
php artisan migrate
php artisan vendor:publish # and choose Backpack/CRUD, in my case it was number 2
```

File: app/Http/Controllers/Admin/MonsterCrudController.php [hunk 1]
```diff
@@ -1217,6 +1218,14 @@ public static function getFieldsArrayForUploadsTab()
...
         ];
 
+        $fields[] = [ // dropzone
+            'label' => 'Dropzone'.backpack_pro_badge(),
+            'name'  => 'dropzone',
+            'type'  => 'dropzone',
+            'hint'  => 'Some info',
+            'tab'   => 'Uploads',
+        ];
+
```
Comment (tabacitu) [inline app/Http/Controllers/Admin/MonsterCrudController.php:1228]: I don't see any events here 👀 Right now we're not moving the file from the temporary directory to the final directory, correct?

Comment (tabacitu) [reply app/Http/Controllers/Admin/MonsterCrudController.php:1228]: Example here - https://backpackforlaravel.com/docs/5.x/release-notes#define-model-events-directly-on-fields

CleanShot 2023-02-07 at 20 14 15@2x

Comment (tabacitu) [reply app/Http/Controllers/Admin/MonsterCrudController.php:1228]: Example event:
```php
        $fields[] = [ // dropzone
            'label' => 'Dropzone'.backpack_pro_badge(),
            'name'  => 'dropzone',
            'type'  => 'dropzone',
            'hint'  => 'Some info',
            'tab'   => 'Uploads',
            'events' => [
               'saving' => (function($entry) {
                    // Step 1. Get the temporary file path.
                    
                    // Step 2. Move that temp file to the final directory & rename it.
                    
                    // Step 3. Save the final path on the entry attribute.
                    $entry->image = $final_path;
                })
            ]
        ];
```

**Comment by tabacitu** [comment]
Time to polish this PR (or create another one) towards the v6 branch of the demo, right? I'll let you guys @pxpm and @jorgetwgroup decide who does this and who tests it.

**Comment by pxpm** [comment]
It would be more work to clean this PR than creating a new one. So ... Bye and See you Soon! 👍

**Pull request closed without merging**